scripts/cli.sh

```diff
@@ -16,7 +16,7 @@ write_to_file() {
     printf "# %s\n\n\`\`\`\n%s\n\`\`\`" "$program" "$cmd" > "$file"
 
     # Adjust the width of the help text and append to the end of file
-    sed -i -e '$a\'$'\n''\n''<style> .content main {max-width:88%;} </style>' "$file"
+    printf '%s\n\n' '' '<style> .content main {max-width:88%;} </style>' >> "$file"
```

Comment:
```suggestion
    printf '%s\n' '' '<style> .content main {max-width:88%;} </style>' >> "$file"
```

I think we might want to remove the 2nd newline. The markdown linter isn't happy